Search suggestion system/README.md

```
@@ -0,0 +1,64 @@
+
+**2. Run the Program:**
+After compiling, run the program:
+
+```bash
+java SearchSuggestions
+```
+
+**3. Example Session:**
+The program will first load the words from `dictionary.txt`.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)